examples/api/lib/material/text_button/text_button.0.dart

```diff
@@ -6,78 +6,473 @@ import 'package:flutter/material.dart';
 
 /// Flutter code sample for [TextButton].
 
-void main() => runApp(const TextButtonExampleApp());
+void main() {
+  runApp(const TextButtonExampleApp());
```

Comment:
An example test is needed
https://github.com/flutter/flutter/tree/master/examples/api#writing-tests

Reply:
I was just adding one when I discovered that the testing framework doesn't like my bonkers URLs. Adding the images to the asssets-for-api-docs repo now, then I'll upload a simple test.

Reply:
Done.